test/integration/consumer_group_test.exs

```diff
@@ -211,4 +211,16 @@ defmodule KafkaEx.ConsumerGroup.Test do
 
     assert 0 >= offset_fetch_response_offset
   end
+
+  test "can join a consumer group" do
+    random_group = generate_random_string
+    random_topic = generate_random_string
+    KafkaEx.create_worker(:join_group, [uris: uris, consumer_group: random_group])
+    KafkaEx.produce(%Proto.Produce.Request{topic: random_topic, partition: 0, required_acks: 1, messages: [%Proto.Produce.Message{value: "hey"}]})
```

Comment:
Heh. Good catch. Needless to say, "foo" and "bar" existing in my integration environment :-). Will double check and fix either way.